cuda_core/pyproject.toml

```diff
@@ -43,6 +43,8 @@ classifiers = [
 ]
 dependencies = [
     "numpy",
+    # TODO: change this once cuda-bindings is packaged, see NVIDIA/cuda-python#105
+    "cuda-python <=12.7.*",
```

Comment:
I think this should instead be "=12.*". (or ">= 12, < 13"?)

These bindings follow CUDA's backwards compatibility grantees, so future minor versions are expected to would seamlessly. Furthermore, this lets bindings avoid having to make patch releases for older minor releases; Patch release only for latest 11.x and 12.x.

Reply:
Actually just "<13" since we promise that a single cuda-core supports both 11.x and 12.x bindings.

Reply:
let's do that
```suggestion
    "cuda-python <13",
```